docs/release/signing.md

```diff
@@ -16,30 +16,66 @@ $ rpm --checksig -v <NAME.RPM>
 For example:
 
 ```console
-$ rpm --checksig -v globus-core-8.0-2.osg.x86_64.rpm 
+$ rpm --checksig -v globus-core-8.0-2.osg.x86_64.rpm
 globus-core-8.0-2.osg.x86_64.rpm:
     Header V3 DSA signature: OK, key ID 824b8603
     Header SHA1 digest: OK (2b5af4348c548c27f10e2e47e1ec80500c4f85d7)
     MD5 digest: OK (d11503a229a1a0e02262034efe0f7e46)
     V3 DSA signature: OK, key ID 824b8603
 ```
 
-The OSG Packaging Signing Key
------------------------------
+The OSG Packaging Signing Keys
+------------------------------
 
-|              |                                                        |
-|--------------|--------------------------------------------------------|
-| Location     | `/etc/pki/rpm-gpg/RPM-GPG-KEY-OSG`                     |
-| Download     | [UW-Madison](https://vdt.cs.wisc.edu/RPM-GPG-KEY-OSG), [GitHub](https://raw.githubusercontent.com/opensciencegrid/docs/master/docs/release/RPM-GPG-KEY-OSG)  |
-| Fingerprint  | `6459 !D9D2 AAA9 AB67 A251  FB44 2110 !B1C8 824B 8603` |
-| Key ID       | `824b8603`                                             |
+The OSG Software Team has two GPG keys for signing RPMs;
+the first key is used for packages before the 3.6 release series,
+and the second key is used for packages in the 3.6 release series and afterward.
```

Comment:
Oh interesting, I thought we had to switch over for all of Koji and weren't able to limit it to a specific release series